src/App.js

```diff
@@ -36,20 +37,60 @@ class App extends React.Component {
                dataByStatuses[status].push(card);
             });
 
-            console.log(stateObj);
+            //console.log(stateObj);
 
             this.setState(stateObj);
         } catch (error) {
             alert("Ошибка HTTP: " + error);
         }          
     }
 
+    handleCreateCard = async(description) => {        
+        const urlCreateCard = this.props.urlCreateCard;
```

Comment:
we discussed it: you don't need url links passed through props, just import it where needed